### PR TITLE
suggestions

### PR DESCRIPTION
**Pull request by grolimur**
some suggestions (including modifying text, icons & links) after reading the early version of the document

### CODE REVIEW

File: content/zotero_intro_to.md [hunk 1]
```diff
@@ -242,13 +242,13 @@ Open Zotero and click on the "Save to Zotero" button in your browser to save the
 
 Example : [an article from Journal of Wine Research](https://doi.org/10.1080/09571264.2018.1433140)
 
-**|!| Tips : choose the right source of information and do not procrastinate**
+**💡 Tips : choose the right source of information and do not procrastinate**
```
Comment (fflamerie) [inline content/zotero_intro_to.md:245]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

File: content/zotero_intro_to.md [hunk 2]
```diff
@@ -359,7 +359,7 @@ We will focus on some most useful features as you are starting your library, tha
 * searches -> quick vs. advanced, immediate vs. saved with dynamic update,.
 * notes.
 
-**/!\\** In Zotero, the icon associated with a **collection** is a folder but collections are more like **music playlists** (think of iTunes, Spotify, etc.) than folders in your computer filesystem.
+**⚠️** In Zotero, the icon associated with a **collection** is a folder but collections are more like **music playlists** (think of iTunes, Spotify, etc.) than folders in your computer filesystem.
```
Comment (fflamerie) [inline content/zotero_intro_to.md:362]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

File: content/zotero_intro_to.md [hunk 3]
```diff
@@ -384,7 +384,7 @@ We will take **10 minutes** for this exercise.
 
 ### Going further
 
-**|!| Tip : let Zotero works for you**
+**💡 Tip : let Zotero works for you**
```
Comment (fflamerie) [inline content/zotero_intro_to.md:387]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

File: content/zotero_intro_to.md [hunk 4]
```diff
@@ -458,8 +458,7 @@ Produce a document identical to this text.
 * Retrieve and add to your Zotero library the bibliographic references cited, you should find all of them online.
 * Make sure you have installed APA style.
 
-**|!| Tips**
-
+**💡 Tips**
```
Comment (fflamerie) [inline content/zotero_intro_to.md:461]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

File: content/zotero_intro_to.md [hunk 5]
```diff
@@ -509,7 +508,7 @@ Make sure your Zotero data directory is included in the backups of your computer
 
 ![zotero][zotero] [Zotero Documentation : The Zotero Data Directory](https://www.zotero.org/support/zotero_data)
 
-**/!\\  Syncing is not a recommended strategy for backing up** . Indeed, the Zotero servers only store the most recent version of your library, and it takes just a single (possibly automatic) sync to change the server copy. Syncing is useful to get a copy of your library in multiple devices and to create group libraries, but it is different from backing up.
+**⚠️ Syncing is not a recommended strategy for backing up** . Indeed, the Zotero servers only store the most recent version of your library, and it takes just a single (possibly automatic) sync to change the server copy. Syncing is useful to get a copy of your library in multiple devices and to create group libraries, but it is different from backing up.
```
Comment (fflamerie) [inline content/zotero_intro_to.md:511]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

File: content/zotero_intro_to.md [hunk 6]
```diff
@@ -292,7 +292,7 @@ Example : the ISBN of a book  or a report, the DOI of a scientific paper.
 
 If you already have stored PDFs on your computer, it might be possible to retrieve metadata from them afterwards.
 
-**/!\\ This is not the recommended default method**.  It is useful for processing a bunch of PDFs you have in stock, but this should not be your usual method of adding content to your Zotero library.
+**⚠️ This is not the recommended default method**.  It is useful for processing a bunch of PDFs you have in stock, but this should not be your usual method of adding content to your Zotero library.
```
Comment (fflamerie) [inline content/zotero_intro_to.md:295]: to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

**Review by fflamerie — CHANGES_REQUESTED** [review]

File: content/zotero_intro_to.md (hunk 5 above)
Comment (grolimur) [reply content/zotero_intro_to.md:511]: Ok

File: content/zotero_intro_to.md (hunk 6 above)
Comment (grolimur) [reply content/zotero_intro_to.md:295]: Ok

File: content/zotero_intro_to.md (hunk 4 above)
Comment (grolimur) [reply content/zotero_intro_to.md:461]: Ok

**Comment by fflamerie** [comment]
Thank you very much for all these relevant suggestions! I would accept all of them except regrading icons, as I have to fix the emoji conversion issue before.

File: content/zotero_intro_to.md (hunk 3 above)
Comment (grolimur) [reply content/zotero_intro_to.md:387]: Ok

File: content/zotero_intro_to.md (hunk 2 above)
Comment (grolimur) [reply content/zotero_intro_to.md:362]: Ok

File: content/zotero_intro_to.md (hunk 1 above)
Comment (grolimur) [reply content/zotero_intro_to.md:245]: Ok

**Pull request merged**